src/mono/wasm/Wasm.Build.Tests/Blazor/MiscTests.cs

```diff
@@ -98,23 +98,4 @@ public void BugRegression_60479_WithRazorClassLib()
 
         Assert.Contains(bootJson.resources.lazyAssembly.Keys, f => f.StartsWith(razorClassLibraryName));
     }
-
-    
-    [Fact]
-    public async Task TestOverrideHtmlAssetPlaceholders()
-    {
-        var config = Configuration.Release;
-        string extraProperties = "<OverrideHtmlAssetPlaceholders>true</OverrideHtmlAssetPlaceholders>";
```

Comment:
Do we have a test with non-default `OverrideHtmlAssetPlaceholders=false`? This one could serve for this purpose.